### ISSUE TITLE
DST Staging Review: Insufficient color contrast on focus/hover states

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#2833]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** Button Icon
**Staging Review ticket:** [Staging Review: Design System Button Icon Component](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82571)

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**This is an issue with the:** code
**High-priority:** yes
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the [Delete](https://design.va.gov/storybook/?path=/docs/components-va-button-icon--change-file#delete) and [Cancel](https://design.va.gov/storybook/?path=/docs/components-va-button-icon--change-file#cancel) button types, the `--vads-color-error` text color against the `--vads-color-secondary-lightest` background color has a [contrast ratio of 3.59:1](https://webaim.org/resources/contrastchecker/?fcolor=D54309&bcolor=F8DFE2), which doesn't meet WCAG AA requirements.

## Recommended action
A darker red on `:hover` and `:focus` should solve this. Changing the text color to `--vads-color-secondary-dark` for those states gets you to a [5.53:1 ratio](https://webaim.org/resources/contrastchecker/?fcolor=B50909&bcolor=F8DFE2) and passes AA. Bump it up to `--vads-color-secondary-darkest` and you're at a [7.76:1 ratio](https://webaim.org/resources/contrastchecker/?fcolor=B50909&bcolor=F8DFE2) that passes AAA.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 1.4.3 Contrast (Minimum) - Level AA](https://www.w3.org/TR/WCAG21/#contrast-minimum)
- **Modality:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2833?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @nickjg231 @powellkerry @rmessina1010 @rsmithadhoc

**Comment by danbrady** [comment]
There ares difference in the colors that Figma has vs. code. However, as mentioned in the recommendations above, since some colors were so close to achieving AAA, I've tweaked them all slightly. Below are the Figma annotations or [find them here](https://www.figma.com/design/60tl39UqLygUtaTYmuH49k/Icon-Button-Annotations?node-id=0%3A1&t=gm1MRzoe5RcQpmRU-1):

![image](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/381190/1c50e197-6e1c-42ae-b19f-4d20a73a1485)

cc: @briandeconinck
